cfme/infrastructure/config_management/__init__.py

```diff
@@ -271,26 +252,17 @@ class ConfigManager(Updateable, Pretty, Navigatable):
     type = None
     refresh_flash_msg = 'Refresh Provider initiated for 1 provider'
 
-    def __init__(self, name=None, url=None, ssl=None, credentials=None, key=None, appliance=None):
-        Navigatable.__init__(self, appliance=appliance)
-        self.name = name
-        self.url = url
-        self.ssl = ssl
-        self.credentials = credentials
-        self.key = key or name
+    name = attr.ib(default=None)
+    url = attr.ib(default=None)
+    credentials = attr.ib(default=None)
+    key = attr.ib(default=None)
+
```

Comment:
`ssl` attribute in Args doc block but not declared.

Reply:
nice catch, fixed